lib/hof-standalone.js

```diff
@@ -75,11 +75,9 @@ StandaloneController.prototype.saveValues = function saveValues(req, res, callba
 
 // Check if we are editing a previous submission
 StandaloneController.prototype.validate = function validate(req, res, callback) {
-  if (req.query.edit === undefined) {
-    Parent.prototype.validate.apply(this, arguments);
-  } else {
-    callback(new ErrorClass(null, { type: 'EDIT' }, req));
-  }
+  return req.query.edit === undefined
```

Comment:
Sorry disagree with this one, I think the if/else version is much easier to read